vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileFragment.kt

```diff
@@ -289,12 +289,21 @@ class RoomMemberProfileFragment @Inject constructor(
     }
 
     private fun handleShareRoomMemberProfile(permalink: String) {
-        startSharePlainTextIntent(
-                fragment = this,
-                activityResultLauncher = null,
-                chooserTitle = null,
-                text = permalink
-        )
+        val inflater = layoutInflater
```

Comment:
this is a bit useless, why not using directly `layoutInflater` in the line below?

Reply:
I think that's just leftover from experimenting. Will change.